app/models/post_mover.rb

```diff
@@ -91,6 +91,12 @@ def create_first_post(post)
   def move(post)
     @first_post_number_moved ||= post.post_number
 
+    post.replies.each do |reply|
+      reply.raw = reply.raw.sub(/post:[1-9]+/, "post:#{@move_map[post.post_number]}")
```

Comment:
it should be gsub, but really this is far too wide a catch, what if the number is missing from @move_map? what if it happens not to be a quote and just some other part of the post